playwright/pages/labs.ts

```diff
@@ -28,5 +28,8 @@ export class Labs {
         await this.page.evaluate((feature) => {
             window.localStorage.setItem(`mx_labs_feature_${feature}`, "true");
         }, feature);
+        // Reload page so that this applies
+        await this.page.reload();
+        await this.page.waitForSelector(".mx_MatrixChat", { timeout: 30000 });
```

Comment:
You're right, thanks!
This method is a bit of a footgun the way it's currently written because if you call this method within a `beforeAll` that uses the user fixture, your feature may not always be enabled without a reload. 
We should either conditionally reload the page if we're logged in or replace this method with a fixture that we can use as:
```ts
test.use({enableLabsFeatures: ["feature_sliding_sync", "feature_notifications"]});
```
wdyt?

Reply:
@MidhunSureshR I like that idea, it makes the tests cleaner IMO